### PR TITLE
fix(checker): fix transition to ERROR state when duplicating metrics

### PR DESCRIPTION
**Pull request by almostinf**
fix transition to ERROR state when duplicating metrics

Replaced the ERROR condition with EXCEPTION for duplicate names in `checker/check.go`, fixed the corresponding tests

Also fixed 2 linter errors in `database/redis` and `senders/pagerduty`

### CODE REVIEW

File: senders/pagerduty/send.go [hunk 1]
```diff
@@ -17,7 +18,7 @@ const summaryMaxChars = 1024
 // SendEvents implements Sender interface Send
 func (sender *Sender) SendEvents(events moira.NotificationEvents, contact moira.ContactData, trigger moira.TriggerData, plots [][]byte, throttled bool) error {
 	event := sender.buildEvent(events, contact, trigger, plots, throttled)
-	_, err := pagerduty.ManageEvent(event)
+	_, err := pagerduty.ManageEventWithContext(context.TODO(), event)
```
Comment (Tetrergeru) [inline senders/pagerduty/send.go:21]: I think we shouldn't merge `context.TODO()` in master.

Let's use `context.Backgound()` here for the time being

Comment (almostinf) [reply senders/pagerduty/send.go]: Fixed

**Review by Tetrergeru — APPROVED** [review]

**Pull request merged**